Geometry/CaloGeometry/src/CaloGenericDetId.cc

```diff
@@ -130,12 +121,15 @@ std::ostream& operator<<(std::ostream& s, const CaloGenericDetId& id)
   if (id.det() == DetId::Hcal) { 
     std::cerr << "No support for HB/HE/HO/HF in CaloGenericDetId" << std::endl;
     throw cms::Exception("No support");
-  } 
+  }
+  else if(id.isCaloTower()) {
+    std::cerr << "No support for CaloTower in CaloGenericDetId" << std::endl;
+    throw cms::Exception("No support");
+  }
 
    return ( id.isEB() ? s<<EBDetId( id ) :
 	    ( id.isEE() ? s<<EEDetId( id ) :
-	      ( id.isES() ? s<<ESDetId( id ) :
-		( id.isCaloTower() ? s<<CaloTowerDetId( id ) :
-		    ( id.isZDC() ? s<<HcalZDCDetId( id ) :
-		      s<<"UnknownId="<<std::hex<<id.rawId()<<std::dec ) ) ) ) ) ;
+	       ( id.isES() ? s<<ESDetId( id ) :
+		 ( id.isZDC() ? s<<HcalZDCDetId( id ) :
```

Comment:
@ianna - It's right here, it just shows up as moved in the diff because the spacing changed when the CaloTower line was deleted.